packages/zosfiles/src/methods/upload/Upload.ts

```diff
@@ -854,8 +854,8 @@ export class Upload {
         };
     }
 
-    private static async uploadFile(localPath: string, ussPath: string,
-        session: AbstractSession, options: IUploadOptions) {
+    public static async uploadFile(session: AbstractSession, localPath: string, ussPath: string,
```

Comment:
is it possible for this change of session in the parameter order to disrupt anything else in the codebase? @t1m0thyj  (gonna wait for response before approving pr @jace-roell)

Reply:
This is a good catch, because we do need to be careful about changing the order of parameters.

In this case it should be OK because the method was private before, so no one was able to use it until now that we've changed it to be public 😋